### DISCUSSION TITLE
Installazione della versione 1.5.8.

### DISCUSSION THREAD

**Discussion by luigiadoc** [iisgiua/giuaschool#706] [Richieste di aiuto]
Cordiale Antonello @trinko ,
ho notato la pubblicazione della nuova versione del registro Giua.
Posso procedere con l'installazione o è preferibile attendere ?
Grazie anticipatamente.

**Comment by trinko** [comment]
Si, puoi installarla. Per sicurezza fai sempre il backup prima di procedere all'aggiornamento.
